src/services/application.js

```diff
@@ -2,6 +2,13 @@ const CompanyApplication = require("../models/CompanyApplication");
 const CompanyApplicationRules = require("../models/CompanyApplication").CompanyApplicationRules;
 const hash = require("../lib/passwordHashing");
 const AccountService = require("./account");
+const EmailService = require("../lib/nodemailer");
+const {
+    NEW_COMPANY_APPLICATION_ADMINS,
+    NEW_COMPANY_APPLICATION_COMPANY,
+    APPROVAL_NOTIFICATION,
+} = require("./emails/companyApplicationApproval");
```

Comment:
No rejection email?

Reply:
I thought about it, but I didn't know what to put there. "You're not good enough" sounded too bad :P

@andrefmrocha could use your input here

Reply:
What use cases are we thinking for rejecting a company application?

Reply:
We could use the reject reason as some sort of message, besides the generic template.

Nevertheless, the notification should still exist and thus the email as well, regardless of not knowing what to say 😛

Reply:
@andrefmrocha I've discussed this with you over PM but the idea is that a company is rejected if the admins don't like their application. For example their motivation is not valid, handling spam that might find the website and apply just because, etc.

Reply:
It would seem that it would be tricky to create a generic email to reject a company since we have so many use cases. But probably an email saying they were rejected and forwarding any complaints to NI's email could be interesting. Wdyt @andrefmrocha @imnotteixeira ?

Reply:
Well, it depends... Since the beginning I've always thought of the Reject Reason as an internal thing. It might be really bad PR to have a "bad" reject reason leak in an email by some lousy admin. In the end, it's up to you @andrefmrocha 

The options we have are:
* Use a generic email saying they were rejected (we can give some common causes of rejection such as spam) and if they think they were wrongly rejected to come back to us @ niMail
* Give them the actual reject reason in the email (must be well written every time, so it involves a bigger responsibility when reviewing)

Reply:
I agree that the second option can only be taken if there's a review of such email. 

But perhaps this is not worth the hassle, at least for now. Why don't we go with a generic email for now? Less implementation overhead and overall less risky.

Reply:
+1 on @imnotteixeira's first option. This way we have a consistent "face" to show to the companies. If this was a website aimed at a broader / less "serious" audience, I'd advocate for including the reject reason, as many websites do. However, what you mentioned made me prefer the other option.

As an "intermediate" alternative (for later, as it would be more effort and requires handling things that are not yet implemented) we could have some predefined reject reasons and have a generic email that includes them as simple snippets, based on what the operators chose. If the "other" reason is chosen a text field can be filled for internal consumption and the message would just be the generic part. This might be worth implementing later on when the project is already up and running, so as to provide more feedback to companies.